cli/cmd/org/rename.go

```diff
@@ -26,65 +24,81 @@ func RenameCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
-			fmt.Println("Warn: Renaming an org would invalidate dashboard URLs")
-
-			if !cmd.Flags().Changed("org") && ch.Interactive {
-				orgNames, err := OrgNames(ctx, ch)
-				if err != nil {
-					return err
+			resp, err := client.GetOrganization(ctx, &adminv1.GetOrganizationRequest{Name: name})
+			if err != nil {
+				if st, ok := status.FromError(err); ok && st.Code() == codes.NotFound {
+					ch.PrintfError("org %q doesn't exist, run 'rill org list' to see available orgs", name)
+					return nil
 				}
+				return err
+			}
+			org := resp.Organization
 
-				name, err = cmdutil.SelectPrompt("Select org to rename", orgNames, "")
+			// Build update request
+			req := &adminv1.UpdateOrganizationRequest{Name: org.Name}
+
+			if req.NewName != nil {
```

Comment:
`req.NewName` has not been assigned yet (happens on line 55), so this check is ineffectual.

I think if the `if` clause here is moved to after the assignments to `req` (around line 67), then things look good